test/test_functional_tensor.py

```diff
@@ -756,34 +766,39 @@ def test_solarize2(device, dtype, config):
 @pytest.mark.parametrize('device', cpu_and_gpu())
 @pytest.mark.parametrize('dtype', (None, torch.float32, torch.float64))
 @pytest.mark.parametrize('config', [{"sharpness_factor": f} for f in [0.2, 0.5, 1.0, 1.5, 2.0]])
-def test_adjust_sharpness(device, dtype, config):
+@pytest.mark.parametrize('channels', [1, 3])
+def test_adjust_sharpness(device, dtype, config, channels):
```

Comment:
This test seems failing: https://app.circleci.com/pipelines/github/pytorch/vision/10743/workflows/fb105be8-f968-493f-9bf4-e3f22251eec5/jobs/819925/tests

Reply:
Well, when I run this test locally with the command:
```
pytest test/test_functional_tensor.py -vvv -k test_adjust_sharpness
```

It shows:
```
========================================================================================== test session starts ==========================================================================================
platform linux -- Python 3.8.11, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /home/vinhloiit/miniconda3/envs/cv/bin/python
cachedir: .pytest_cache
rootdir: /home/vinhloiit/projects/vision, configfile: pytest.ini
plugins: cov-2.12.1, anyio-2.2.0
collected 28414 items / 28354 deselected / 60 selected                                                                                                                                                  

test/test_functional_tensor.py::test_adjust_sharpness[1-config0-None-cpu] PASSED                                                                                                                  [  1%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config0-None-cuda] PASSED                                                                                                                 [  3%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config0-dtype1-cpu] PASSED                                                                                                                [  5%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config0-dtype1-cuda] PASSED                                                                                                               [  6%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config0-dtype2-cpu] PASSED                                                                                                                [  8%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config0-dtype2-cuda] PASSED                                                                                                               [ 10%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-None-cpu] PASSED                                                                                                                  [ 11%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-None-cuda] PASSED                                                                                                                 [ 13%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-dtype1-cpu] PASSED                                                                                                                [ 15%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-dtype1-cuda] PASSED                                                                                                               [ 16%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-dtype2-cpu] PASSED                                                                                                                [ 18%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config1-dtype2-cuda] PASSED                                                                                                               [ 20%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-None-cpu] PASSED                                                                                                                  [ 21%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-None-cuda] PASSED                                                                                                                 [ 23%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-dtype1-cpu] PASSED                                                                                                                [ 25%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-dtype1-cuda] PASSED                                                                                                               [ 26%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-dtype2-cpu] PASSED                                                                                                                [ 28%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config2-dtype2-cuda] PASSED                                                                                                               [ 30%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-None-cpu] PASSED                                                                                                                  [ 31%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-None-cuda] PASSED                                                                                                                 [ 33%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-dtype1-cpu] PASSED                                                                                                                [ 35%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-dtype1-cuda] PASSED                                                                                                               [ 36%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-dtype2-cpu] PASSED                                                                                                                [ 38%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config3-dtype2-cuda] PASSED                                                                                                               [ 40%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-None-cpu] PASSED                                                                                                                  [ 41%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-None-cuda] PASSED                                                                                                                 [ 43%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-dtype1-cpu] PASSED                                                                                                                [ 45%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-dtype1-cuda] PASSED                                                                                                               [ 46%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-dtype2-cpu] PASSED                                                                                                                [ 48%]
test/test_functional_tensor.py::test_adjust_sharpness[1-config4-dtype2-cuda] PASSED                                                                                                               [ 50%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-None-cpu] PASSED                                                                                                                  [ 51%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-None-cuda] PASSED                                                                                                                 [ 53%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-dtype1-cpu] PASSED                                                                                                                [ 55%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-dtype1-cuda] PASSED                                                                                                               [ 56%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-dtype2-cpu] PASSED                                                                                                                [ 58%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config0-dtype2-cuda] PASSED                                                                                                               [ 60%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-None-cpu] PASSED                                                                                                                  [ 61%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-None-cuda] PASSED                                                                                                                 [ 63%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-dtype1-cpu] PASSED                                                                                                                [ 65%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-dtype1-cuda] PASSED                                                                                                               [ 66%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-dtype2-cpu] PASSED                                                                                                                [ 68%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config1-dtype2-cuda] PASSED                                                                                                               [ 70%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-None-cpu] PASSED                                                                                                                  [ 71%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-None-cuda] PASSED                                                                                                                 [ 73%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-dtype1-cpu] PASSED                                                                                                                [ 75%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-dtype1-cuda] PASSED                                                                                                               [ 76%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-dtype2-cpu] PASSED                                                                                                                [ 78%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config2-dtype2-cuda] PASSED                                                                                                               [ 80%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-None-cpu] PASSED                                                                                                                  [ 81%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-None-cuda] PASSED                                                                                                                 [ 83%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-dtype1-cpu] PASSED                                                                                                                [ 85%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-dtype1-cuda] PASSED                                                                                                               [ 86%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-dtype2-cpu] PASSED                                                                                                                [ 88%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config3-dtype2-cuda] PASSED                                                                                                               [ 90%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-None-cpu] PASSED                                                                                                                  [ 91%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-None-cuda] PASSED                                                                                                                 [ 93%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-dtype1-cpu] PASSED                                                                                                                [ 95%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-dtype1-cuda] PASSED                                                                                                               [ 96%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-dtype2-cpu] PASSED                                                                                                                [ 98%]
test/test_functional_tensor.py::test_adjust_sharpness[3-config4-dtype2-cuda] PASSED                                                                                                               [100%]
```

Reply:
I could not repro neither locally from my side using the same pytorch. According to the failure report, the difference is
```
AssertionError: Tensor-likes are not equal!

Mismatched elements: 56 / 288 (19.4%)
Greatest absolute difference: 5.960464477539063e-08 at index (0, 5, 4)
Greatest relative difference: 1.47366798466619e-07 at index (0, 2, 4)
```

After debugging on Circle CI, max abs error is about 5.9605e-08 for all failing cases. Looks like a flakyness on this particular infrastructure

I propose to do the following fix : 
```diff
# vision/test/common_utils.py
- assert_equal = functools.partial(torch.testing.assert_close, rtol=0, atol=0)
+ assert_equal = functools.partial(torch.testing.assert_close, rtol=0, atol=1e-6)
```